scores-src/src/common/sports/football/index.tsx

```diff
@@ -338,6 +338,7 @@ export function RenderScore(props: { state: State }) {
       </Title>
       <div>
         <RenderClock
+          key={props.state.clock.state}
```

Comment:
Note to self: this may need to also include timeLastStartedOrStopped (undo demo clock refresh weirdness)